### ISSUE TITLE
high resolution image for printing?

### ISSUE THREAD

**Issue by thiennn** [dotnet-presentations/dotnetconf2018#3]
@BethMassi 

We are preparing for a .NET Conf local event. Could you share us some high resolution design material so that we can use them in our backdrop and standee?

Thanks

**Comment by BethMassi** [comment]
@thiennn Did you not find what you need in the creative folder? What resolution and format are you looking for?

**Comment by thiennn** [comment]
No, I didn't found a good material to used for our standee. It would be best if you can share the .psd files, or some background images (resolution 300 pixels/inch) so that we can put our custom content on top of it
